Gemfile

```diff
@@ -11,10 +11,11 @@ def location_for(place, fake_version = nil)
 end
 
 group :test do
-  gem 'voxpupuli-test', '>= 1.4.0',  :require => false
+  gem 'voxpupuli-test', '< 2.0.0',   :require => false
   gem 'coveralls',                   :require => false
   gem 'simplecov-console',           :require => false
-  gem 'rspec-puppet-facts',          :require => false, :git => 'https://github.com/mcanevet/rspec-puppet-facts', :ref => '9541292d4fc35db3be1badace673c1108154b571'
+  gem 'rspec-puppet-facts', '2.0.0', :require => false
+  gem 'facterdb', '1.4.0',           :require => false
```

Comment:
The next modulesync will overwrite this file, but these are the versions we'll end up with.  I needed to get them in here for the build not to hit the 50 minute travis timeout.